packages/babel-plugin/src/utils/traverse-expression/traverse-call-expression.ts

```diff
@@ -50,6 +50,10 @@ export const traverseCallExpression = (
       // Right now we are only supported these 2 flavors. If we have complex case like `func('arg').fn().variable`,
       // it will not get evaluated.
       if (t.isIdentifier(callee)) {
+        if (callee.name === 'css' && t.isObjectExpression(expression.arguments[0])) {
```

Comment:
css might be aliased?
```
import { css as css2 } from "@compiled/react"

css2(...)
```
Also, css might have multiple arguments?
```
css({
   color: 'red'
}, {
   background: 'red'
})
```

Reply:
Both very good points! 

- [ ] css might be aliased
- [ ] css might have multiple arguments